server/chat-plugins/rock-paper-scissors.ts

```diff
@@ -0,0 +1,406 @@
+/**
+ * Rock, Paper, Scissors plugin by Mia
+ * @author mia-pi-git
+ */
+import {Utils} from '../../lib/utils';
+
+/** Map<to, [from, rounds]> */
+const challengeRequests: Map<string, string> = new Map();
```

Comment:
`Map<to, [from, rounds]>` does not match the actual signature specified here.

Also, I would call this `challenges`.

Reply:
You might also want to persist this across hotpatches with the new `oldPlugins` API.

Reply:
I would also put this below your allcaps constants.